### PR TITLE
Readme cleanup and added contributing file

### PR DESCRIPTION
**Pull request by trishahanlon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,52 +2,75 @@
...
+--- 
+
+## Optimizely + Cloudflare Worker Template
+
+> Starter Kit for running Optimizely Full Stack feature flags and experiments on Fastly's Compute@Edge offering.
+
+
+The Optimizely worker template for Cloudflare workers embeds and extends our [JavaScript SDK](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/javascript-node-sdk) to provide a starting point for you to implement experimentation and feature flagging for your experiences at the edge. For a guide to getting started with our platform more generally, this can be combined with the steps outlined in our [JavaScript Quickstart](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/javascript-node-quickstart). 
 
 ### Caching with cloudflare
```
Comment (mikechu-optimizely) [inline README.md]: Should `cloudflare` be capitalized `Cloudflare`?

File: README.md [hunk 2]
```diff
@@ -2,52 +2,75 @@
...
 
-6. To deploy the worker on cloudflare.
+6. Deploy the worker on Cloudflare.
 
    ```
    wrangler publish
    ```
 
-7. To optionally tail the logs for debugging when accessing worker deployed on cloudflare.
+7. Optionally tail the logs for debugging when accessing worker deployed on Cloudflare.
```
Comment (mikechu-optimizely) [inline README.md]: NIT: "Optionally, tail the..."

File: README.md (hunk 1 above)
Comment (trishahanlon) [reply README.md]: Yup, fixed! Also apparently Workers is capitalized also. Good call!

**Review by opti-jnguyen — APPROVED** [review]
LGTM!

**Review by opti-jnguyen — APPROVED** [review]

**Pull request merged**